common/scala/src/main/scala/whisk/core/entity/Exec.scala

```diff
@@ -210,4 +230,7 @@ protected[core] object Exec
             }
         }
     }
+
+    val whiskConfig = new WhiskConfig(Map(WhiskConfig.entityActioncodeLimit -> "50m"))
```

Comment:
We need to come up with a solution that doesn't need to create a new `WhiskConfig` instance.